pyproject.toml

```diff
@@ -1,6 +1,5 @@
 [build-system]
 requires = [
-    "setuptools >= 40.8",
     "wheel",
 ]
 build-backend = "setuptools.build_meta"
```

Comment:
Have you considered having maybe a completely separate backend?

Reply:
I have. Maybe something that vendors all of the dependencies or just hard-codes the behavior especially for setuptools. It's not out of consideration yet.

Reply:
Yep, that's exactly what I had in my mind.